code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -63,6 +63,30 @@
 	item_color = "clown"
 	var/footstep = 1	//used for squeeks whilst walking
 	shoe_sound = "clownstep"
+	var/enabled_waddle = TRUE
+	var/datum/component/waddle

```

Comment:
I don't think this var is actually called anywhere in the code, so it can be removed right?

Reply:
Need to change the code to use the holder, another good catch thanks!